public docs for ref chart (#9305)
circleci/circleci-docs: jekyll/_cci2/server/latest/operator/reference-monitoring-stack.adoc

---
contentTags:
  platform:
    - Server v4.8
    - Server Admin
---

> = Server Monitoring Stack Reference Helm Chart

Reviewer: ```suggestion
= Server monitoring stack reference Helm chart
```

:page-layout: classic-docs
:page-liquid:

> :page-description: Learn how to deploy your own monitoring stack using our helm chart as a reference.

Reviewer: ```suggestion
:page-description: Learn how to deploy your own monitoring stack using our Helm chart as a reference.
```

:icons: font
:toc: macro
:toc-title:

> CircleCI server customers want to know what metrics are available for gaining insight into the performance of their builds. While we don't support or provide any tooling to provide observability into the running services and infrastructure CircleCI server operates in, we can share a sample of how observability can be set up using the most common tools via a helm chart package.

Reviewer: Wonder about who we're talking to here, server admins? adding a suggestion to reframe

Author: yes this would be really for server admins. i'll update this

> CircleCI server customers want to know what metrics are available for gaining insight into the performance of their builds. While we don't support or provide any tooling to provide observability into the running services and infrastructure CircleCI server operates in, we can share a sample of how observability can be set up using the most common tools via a helm chart package.

Reviewer: ```suggestion
You may want to access metrics to gain insight into the performance of builds on your CircleCI server installation. This page provides a sample to show how observability can be set up using the most common tools via a Helm chart package.

CircleCI does not support or provide any tooling to provide observability into the running services and infrastructure that CircleCI server operates in.
```

> This is however only meant to be a reference or guide on how things could be set up and is not an officially supported product. Our reference monitoring stack currently contains samples on how to set up and configure grafana, prometheus and telegraf as well as some built-in dashboards for monitoring key service level inidicators.

Reviewer: ```suggestion
This guide is provided as a reference on how you could  set up observability and is not an officially supported product. Our reference monitoring stack currently contains samples on how to set up and configure Grafana, Prometheus, and Telegraf, as well as some built-in dashboards for monitoring key service level indicators.
```

> You can find the link:https://github.com/CircleCI-Public/circleci-server-monitoring-reference?tab=readme-ov-file#server-monitoring-stack[reference monitoring stack on our public github] as well as more information on how you could configure it's setup. You may also find information on link:https://github.com/CircleCI-Public/circleci-server-monitoring-reference?tab=readme-ov-file#modifying-or-adding-grafana-dashboards[adding to or editing the existing dashboards] here.

Reviewer: ```suggestion
You can find the link:https://github.com/CircleCI-Public/circleci-server-monitoring-reference?tab=readme-ov-file#server-monitoring-stack[reference monitoring stack on our public GitHub], as well as more information on how you could configure this setup. You will also find information on link:https://github.com/CircleCI-Public/circleci-server-monitoring-reference?tab=readme-ov-file#modifying-or-adding-grafana-dashboards[adding to or editing the existing dashboards].
```
